_guide/_pages/css/linting.md

```diff
@@ -8,19 +8,11 @@ The styleguide provides a method of linting [Sass] code to ensure it conforms
 to the rules in the styleguide. This linting tool will go through all Sass code
 and issue warnings wherever the code differs from the styleguide. We've created
 a specific [`.scss-lint.yml` file][scss-lint yaml] that's configured to work
-with the css coding styleguide. There are three ways to setup linting:
+with the css coding styleguide. There are two ways to setup linting:
```

Comment:
It existed previously, but I believe "setup" here in the verb form should be two words:

```suggestion
with the css coding styleguide. There are two ways to set up linting:
```

https://www.merriam-webster.com/dictionary/setup#note-1